Formula/ansible.rb

```diff
@@ -268,8 +268,8 @@ class Ansible < Formula
   end
 
   resource "openstacksdk" do
-    url "https://files.pythonhosted.org/packages/94/47/76d19285d0f4a631c1b01bec39f4cccb7fc613d23bcd5a2785012346a290/openstacksdk-1.0.1.tar.gz"
-    sha256 "365e5dcca64e16e74a4f9f9d2a1f2207970e946d9a5c482549cfa5ec02ca2c98"
+    url "https://files.pythonhosted.org/packages/e9/3b/560ac295eb3db0cc03fd78b46bbaf322103e460bde06ded056a3dfb3c585/openstacksdk-1.1.0.tar.gz"
+    sha256 "cd4d1fc523228e0cb9308000f34be7c204eab14bd278ddc116eefa47d68d7496"
   end
```

Comment:
except this is the version update, the others are just resource name updates, do we really need this PR?

Reply:
This one doesn't look required -- as you may have seen, we've now updated brew-pip-audit to include which package upgrades are actually fixing security issues.